src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1159,127 +1162,109 @@ public void updateInventory()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		incrementStatistic(statistic, material, 1);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic, @NotNull Material material)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		decrementStatistic(statistic, material, 1);
 	}
 
 	@Override
 	public int getStatistic(@NotNull Statistic statistic, @NotNull Material material)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return statistics.getStatistic(statistic, material);
 	}
 
 	@Override
 	public void incrementStatistic(@NotNull Statistic statistic, @NotNull Material material, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, material, statistics.getStatistic(statistic, material) + amount);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic, @NotNull Material material, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, material, statistics.getStatistic(statistic, material) - amount);
 	}
 
 	@Override
 	public void setStatistic(@NotNull Statistic statistic, @NotNull Material material, int newValue)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, material, newValue);
 	}
 
 	@Override
 	public void incrementStatistic(@NotNull Statistic statistic, @NotNull EntityType entityType)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		incrementStatistic(statistic, entityType, 1);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic, @NotNull EntityType entityType)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		decrementStatistic(statistic, entityType, 1);
 	}
 
 	@Override
 	public int getStatistic(@NotNull Statistic statistic, @NotNull EntityType entityType)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return statistics.getStatistic(statistic, entityType);
 	}
 
 	@Override
 	public void incrementStatistic(@NotNull Statistic statistic, @NotNull EntityType entityType, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, entityType, statistics.getStatistic(statistic, entityType) + amount);
 	}
 
 	@Override
 	public void decrementStatistic(@NotNull Statistic statistic, @NotNull EntityType entityType, int amount)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		statistics.setStatistic(statistic, entityType, statistics.getStatistic(statistic, entityType) - amount);
```

Comment:
two operations for what can be done in one, I suggest you to add a `increment` and `decrement` method in your `StatisticsMock` and implement the logic there.